.vscode/settings.json

```diff
@@ -8,6 +8,8 @@
     "cloc",
     "COSE",
     "dealloc",
+    "Decapsulates",
+    "decapsulation",
```

Comment:
though: I had no idea this was an actual english word :D

Reply:
https://en.wikipedia.org/wiki/Key_encapsulation_mechanism

The intent here is to establish distinct terminology instead of just "encrypt / decrypt" everywhere, since the meaning for the latter is too broad..

Reply:
Oh it's even convention in this context, nice 👍